packages/bpk-component-image/src/BpkBackgroundImage.js

```diff
@@ -158,9 +162,10 @@ class BpkBackgroundImage extends Component<BpkBackgroundImageProps> {
 }
 
 BpkBackgroundImage.propTypes = {
-  height: PropTypes.number.isRequired,
+  aspectRatio: widthHeightAspectRatioPropType,
+  height: widthHeightAspectRatioPropType,
```

Comment:
Can we use the `deprecated` util from bpk-react-utils for these here?

Something like 

```
height: deprecated(PropTypes.number, 'Use "aspectRatio" instead.'),
```

Which would throw warnings for consumers and get them to move over and action sooner

Reply:
Good idea!